pdns/dnsrulactions.hh

```diff
@@ -126,7 +126,7 @@ public:
   }
   bool matches(const DNSQuestion* dq) const override
   {
-    return dq->dh->cd || (getEDNSZ((const char*)dq->dh, dq->len) & EDNS_HEADER_FLAG_DO);    // turns out dig sets ad by default..
+    return dq->dh->ad || (getEDNSZ((const char*)dq->dh, dq->len) & EDNS_HEADER_FLAG_DO);
```

Comment:
ohw, this file is not part of the recursor. Based on the documentation, we should make this line
```
return dq->dh->cd || dq->dh->ad ||(getEDNSZ((const char*)dq->dh, dq->len) & EDNS_HEADER_FLAG_DO);
```

As any of these flags could mean "do something with DNSSEC"